book/listings/css/2/style.css

```diff
@@ -0,0 +1,3 @@
+button.text-button {
```

Comment:
`.text-button` is a gtk style class, you really don't want to style anything like this. Add your own name or style class instead. It's too generic - and you're probably picked `.text-button` because otherwise it affects the window close button, right? :) But it's still too broad.

Reply:
Now that you’ve read the text, do you agree that I don‘t really recommend it, but mostly try to make a point that GTK adds style classes?

Reply:
Yep.